erpnext/stock/doctype/item/item.js

```diff
@@ -795,4 +795,43 @@ frappe.ui.form.on("UOM Conversion Detail", {
 			});
 		}
 	}
-})
+});
+
+
+frappe.tour['Item'] = [
+	{
+		fieldname: "item_name",
+		title: __("Item Name"),
+		description: __("This is the actual name of the product or service.")
+	},
+	{
+		fieldname: "item_code",
+		title: __("Item Code"),
+		description: __("The Item Code can be thought of as a short-form that refers to the item.") + " " +
+			__("For more information click here:") + " " +
+			"<a href='https://docs.erpnext.com/docs/user/manual/en/stock/articles/item-codification' target='_blank'>Item Codification</a>" +
+			__("."),
+	},
+	{
+		fieldname: "item_group",
+		title: __("Item Group"),
+		description: __("This is used for categorizing items under a common criteria.") + " " +
+			__("For more information click here:") + " " +
+			"<a href='https://docs.erpnext.com/docs/user/manual/en/stock/item-group' target='_blank'>Item Group</a>" +
+			__("."),
+	},
+	{
+		fieldname: "stock_uom",
+		title: __("Default Unit of Measure"),
+		description: __("The Item Code can be thought of as a short-form that refers to the item.") + " " +
```

Comment:
This Description repeation. It could be:

"It's a UoM in which stock balance will be maintained for this Item."

Am further updating Stock Onboarding and Description in this doc. Let's catch-up quickly to discuss further.

https://docs.google.com/document/d/1O2hGLobhVUdzZKq_P_EujfqxDi9pRx_hI0hnuYWdbAk/edit#